govcd/vapp.go

```diff
@@ -1097,17 +1110,54 @@ func (vapp *VApp) AddIsolatedNetwork(newIsolatedNetworkSettings *VappNetworkSett
 
 }
 
+func validateNetworkConfigSettings(networkSettings *VappNetworkSettings) error {
+	if networkSettings.Name == "" {
+		return errors.New("network name is missing")
+	}
+
+	if networkSettings.Gateway == "" {
+		return errors.New("network gateway ip is missing")
```

Comment:
Please change "ip" to "IP" when it is inside text messages.

Reply:
Fixed